client/components/Oodlet.vue

```diff
@@ -1,28 +1,69 @@
 <template>
   <div class="oodlet">
-    <h1>Oodlet</h1>
-    <span class="due-date">Due: </span>{{ dueDate }}
-    <ol>
-      <li v-for="quantifiedThingy in quantifiedThingies">
-        <quantified-thingy :quantifiedThingy="quantifiedThingy"></quantified-thingy>
-      </li>
-    </ol>
-    <button @click="save">Save</button>
-    <button @click="reset">Reset</button>
+    <div>
+      <h1>Oodlet</h1>
+    </div>
+
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <strong>Due in: </strong>
+        <span v-if="dueDate" class="due-in-countdown" :class=dueInClass>{{ dueIn }}</span>
+        <span v-else class="due-in-countdown red">Expired</span>
+      </div>
+      <div class="panel-body">
+        <table class="table table-striped table-hover ">
+          <tbody>
+            <quantified-thingy v-for="quantifiedThingy in quantifiedThingies" :quantifiedThingy="quantifiedThingy"></quantified-thingy>
+          </tbody>
+        </table>
+      </div>
+      <div class="panel-footer" v-show="quantifiedThingies.length > 0">
+        <button class="btn btn-danger push-right" @click="reset">Reset</button>
+      </div>
+    </div>
   </div>
 </template>
 
 <script>
   import QuantifiedThingy from './QuantifiedThingy.vue';
+  import moment from 'moment';
 
-  export default{
+  export default {
+    data() {
+      return {
+        now : moment()
+      }
+    },
     computed: {
-      quantifiedThingies(){
+      quantifiedThingies() {
         return this.$store.getters.quantifiedThingies;
       },
-      dueDate(){
-        return this.$store.getters.dueDate;
+      dueDate() {
+        if(this.$store.getters.dueDate) {
+          return this.$store.getters.dueDate;
```

Comment:
Extract `dueDate` is used as `moment(this.dueDate)` throughout this component, It would be better to return it already transformed using moment.